gradio/interface.py

```diff
@@ -650,75 +635,26 @@ def __repr__(self):
             repr += "\n|-{}".format(str(component))
         return repr
 
-    async def submit_func(self, *args):
-
-        prediction = await self.run_prediction(args)
-        # TODO(faruk): We don't have tuple or array clearence in Blocks, can we remove this and have one standart?
-        return prediction[0] if len(self.output_components) == 1 else prediction
-
     async def run_prediction(
```

Comment:
This looks great! It looks like we don't need this function anymore, thx to the cleanups. Let's use this [function ](https://github.com/gradio-app/gradio/blob/main/gradio/blocks.py#L549)inside Blocks?

Reply:
Good catch! Just to confirm, you mean use `Blocks.call_function` instead, right?